packages/evm/script/solidity/Deploy.s.sol

```diff
@@ -29,8 +29,9 @@ contract CoreDeployer is ScriptUtils {
     }
 
     function _deployCore(address registry) internal returns (address core) {
+        address owner = vm.addr(vm.envUint("BOOST_CORE_OWNER_ADDRESS"));
```

Comment:
This should use envAddr to save the extra conversion https://book.getfoundry.sh/cheatcodes/env-address?highlight=EnvAdd#signature